src/components/SharedComponents/SettingsComponent.vue

```diff
@@ -44,9 +44,9 @@
         <div class="col-md-12" id="scoreSelect">
           <p>Score to Win:
           <select class="custom-select" name="select" v-model="selected">
-            <option value="15">Short (score 15)</option>
-            <option value="25">Medium (score 25)</option>
-            <option value="35">Long (score 35)</option>
+            <option value="25">Short (score 25)</option>
```

Comment:
Did you test these limits? Are they big enough given the new cards?

Reply:
Ive ran through them a few times each. They seem big enough but the short is a little bit longer than short games used to be based on what cards you get. It depends approximately how long we want each mode to be respectively.

Reply:
OK.